src/buildercore/terraform.py

```diff
@@ -91,25 +90,22 @@
 FASTLY_MAIN_VCL_KEY = 'main'
 
 def render(context):
+    template = TerraformTemplate()
     fn_list = [
         render_fastly,
         render_gcs,
         render_bigquery
     ]
-    partial_tf_list = [fn(context) for fn in fn_list]
-
-    def merge(a, b):
-        deepmerge(a, b)
-        return a
+    [fn(context, template) for fn in fn_list]
```

Comment:
(minor) I'd probably change that to a regular loop because we don't use the array (and that may be a bit confusing):

```python
for fn in fn_list:
    fn(context, template)
```

Reply:
Yes, no need for side-effects in list comprehensions